[ION-9594] Custom Log Appender Documentation Must Include Enforcements (#515)
mulesoft/docs-runtime-manager: modules/ROOT/pages/custom-log-appender.adoc

https://logging.apache.org/log4j/2.x/manual/configuration.html#ConfigurationSyntax[Log4j Configuration Syntax].

To enable the Log4j appender, update the `log4j2.xml` configuration file with your logger settings.

> Include the following Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile` as shown in the example below along with your own logger settings.

Reviewer: ```suggestion
Include the Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile` in the example along with your logger settings.
```

> Include the following Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile` as shown in the example below along with your own logger settings.

Reviewer: Can we change these lines like this:

To enable the Log4j appender, update the `log4j2.xml` configuration file with your logger settings and include the Log4j appenders `Log4J2CloudhubLogAppender` and `RollingFile`.
In Anypoint Studio, the `log4j2.xml` file is located in the `src/main/resources` directory.

RollingFile::
Sends log data to the filesystem of the VM.

[WARNING]

> If the appenders mentioned above are not part of the `log4j2.xml` file, Cloudhub support may lose visibility into the logs. It is strongly advised to include the above appenders in your custom configurations.

Reviewer: ```suggestion
You must include the appenders in your custom configurations in the `log4j2.xml` file to enable MuleSoft Support team visibility into the logs.
```
